planutils/package_installation.py

```diff
@@ -114,16 +114,23 @@ def install(targets):
     to_install.reverse()
 
     if to_install:
-        print("\nAbout to install the following packages: %s" % ', '.join(to_install))
+        to_install_desc = ["%s (%s)" % (pkg, PACKAGES[pkg]['size']) for pkg in to_install]
+        print("\nAbout to install the following packages: %s" % ', '.join(to_install_desc))
         if input("  Proceed? [Y/n] ").lower() in ['', 'y', 'yes']:
             installed = []
             for package in to_install:
-                print("Installing %s..." % package)
+                package_path = os.path.join(CUR_DIR, 'packages', package)
+                print("Installing %s..." % package, end='')
                 try:
                     installed.append(package)
-                    subprocess.check_call('./install', cwd=os.path.join(CUR_DIR, 'packages', package))
+                    subprocess.check_call('./install', cwd=package_path)
+                    size = subprocess.check_output('du -sh .',
+                                                   cwd=package_path,
+                                                   shell=True,
+                                                   encoding='utf-8').split('\t')[0]
+                    print("done. (size: %s)" % size)
```

Comment:
If `./install` has any output, the pattern of `installing ... done` will not work. It would be safer to to say "Installing x ...\n" and "Finished installing x." (same for uninstalling, of course)